test/page.spec.js

```diff
@@ -196,6 +196,13 @@ module.exports.addTests = function({testRunner, expect, puppeteer, DeviceDescrip
       const isFive = await page.evaluate(e => Object.is(e, 5), aHandle);
       expect(isFive).toBeTruthy();
     });
+    it('should simulate a user gesture', async({page, server}) => {
+      await page.evaluate(() => {
```

Comment:
add a test of for the string case: page.evaluate('...')

Reply:
done.